File: scripts/vml-out-probe/langs/out.cpp
// out.cpp —— VML 跨语言「输出」判据（期望恰好三行，见 run-langs.sh）
// 写法照 corpus/cpp/skel.* —— 共享库同时提供 println_str / println_int。
void println_str(char* s);
void println_int(int v);
int main() {
    println_str("OUT-STR=abc");
    print_str("OUT-INT=");
    println_int(42);
    println_str("OUT-PUN=hello, world");
    return 0;
}

File: scripts/vml-out-probe/langs/out.f90
// out.f90 —— VML 跨语言「输出」判据（期望恰好三行，见 run-langs.sh）
program p
  print '(A)', 'OUT-STR=abc'
  print '(I0)', 42
  print '(A)', 'OUT-PUN=hello, world'
end program p
